tests/kernel/mem_protect/protection/boards/stm32n6570_dk_stm32n657xx_sb.conf

```diff
@@ -0,0 +1 @@
+CONFIG_USERSPACE=y
```

Comment:
I don't think this is the correct fix.
If Protection test is designed to run w/o USERSPACE, then we should review mpu_regions settings dependencies.

Let's remove this commit from the series in order not to block the other fixes.